lib/services/project-data-service.ts

```diff
@@ -41,6 +42,13 @@ export class ProjectDataService implements IProjectDataService {
 		return projectDataInstance;
 	}
 
+	@exported("projectDataService")
```

Comment:
can you export `getProjectData` as well - it will be used from Sidekick to get paths to app and App_Resources directories